src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -179,6 +185,49 @@ function useSidebarMetadataFetcher(
         [api, file, onApiError],
     );
 
+    const extractSuggestions = React.useCallback(
+        async (templateKey: string, fields: MetadataTemplateField[]) => {
+            const aiAPI = api.getIntelligenceAPI();
+
+            const oldToken = api.xhr.token;
+            api.xhr.token = async function overwrittenToken(...args) {
+                const tokenOjb = await oldToken(...args);
+
+                return {
+                    read: tokenOjb.read,
+                };
+            };
+
+            let answer = {};
+            try {
+                answer = await aiAPI.extractStructured({
+                    items: [file],
+                    metadata_template: {
+                        scope: 'enterprise',
+                        template_key: templateKey,
+                        type: 'metadata_template',
+                    },
+                });
+            } catch (e) {
+                // eslint-disable-next-line no-console
```

Comment:
don't we want to catch and handle this error somehow?  We have existing error code `ERROR_CODE_FETCH_METADATA_SUGGESTIONS` which could be useful